test/Elastic.Apm.Tests/ApiTests/ConvenientApiTransactionTests.cs

```diff
@@ -485,10 +495,8 @@ private async Task<MockPayloadSender> AssertWith1TransactionAsync(Func<ApmAgent,
 			Assert.Equal(TransactionName, payloadSender.Payloads[0].Transactions[0].Name);
 			Assert.Equal(TransactionType, payloadSender.Payloads[0].Transactions[0].Type);
 
-			//`await Task.Delay` is inaccurate, so we enable 10% error in the test
-			var expectedTransactionLength = SleepLength - (SleepLength * 0.1);
 			var duration = payloadSender.Payloads[0].Transactions[0].Duration;
-			Assert.True(duration >= expectedTransactionLength, $"Expected {duration} to be greater or equal to: {expectedTransactionLength}");
+			Assert.True(duration >= SleepLength, $"Expected {duration} to be greater or equal to: {SleepLength}");
```

Comment:
Similar to previous comment, make this an extension method on Assert `Assert.MinimumDuration(duration)`

Reply:
Same here... Assert is a class with static methods. I think we can't extend that.